.github/workflows/api-swagger-ui.yml

```
@@ -0,0 +1,76 @@
+name: Build and Push Swagger-UI to R2 mainnet-api-docs.spacemesh.network
+
+on:
+  release:
+    types: [published]
+    
+jobs:
+  check-version:
+    runs-on: ubuntu-22.04
+    outputs:
+      go-sm-api-version: ${{ steps.go-sm-api-version.outputs.GO_SM_API_VERSION }}
+    steps:
+    - name: Checkout target repository on last release
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: 'refs/tags/${{ github.event.release.tag_name }}'
+    
+    - name: Extract dependency version
+      id: go-sm-api-version
+      run: |
+        version=$(grep -E 'github.com/spacemeshos/api/release/go' go.mod | awk '{print $2}')
```

A dependency can appear multiple times in the `go.mod` file (e.g. when using replacements) a safer command to get the version of a dependency that is used to build the binary would be:

```suggestion
        version=$(go list -m 'github.com/spacemeshos/api/release/go' | awk '{print $2}')
```